### ISSUE TITLE
PlaygroundPrefs error

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#333]
@K-Meech @NicoKiaru 
I have something weird that I do not understand.
In IntelliJ I can compile the latest master of MoBIE.
But when I `mvn clean install` it into Fiji I am getting this error:
```
Caused by: java.lang.NoClassDefFoundError: sc/fiji/bdvpg/PlaygroundPrefs
	at de.embl.cba.mobie.MoBIE.<init>(MoBIE.java:68)
	at de.embl.cba.mobie.command.OpenMoBIEProjectBranchCommand.run(OpenMoBIEProjectBranchCommand.java:25)
	at org.scijava.command.CommandModule.run(CommandModule.java:196)
	at org.scijava.module.ModuleRunner.run(ModuleRunner.java:163)
	at org.scijava.module.ModuleRunner.call(ModuleRunner.java:124)
	... 6 more
Caused by: java.lang.ClassNotFoundException: sc.fiji.bdvpg.PlaygroundPrefs
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 11 more
```
This class exists in bdv-playground 0.2.14-SNAPSHOT, which is the version that also is in the jars folder...

**Comment by tischi** [comment]
I could fix it by updating to `0.2.14` in the pom

**Comment by NicoKiaru** [comment]
SNAPSHOT builds are not reproducible, it's possible that you are not using your local SNAPSHOT but the distant one which has different classes
